src/client/js/partials/exposures-setup.js

```diff
@@ -0,0 +1,58 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/** @type {HTMLDivElement} */
+// @ts-ignore: We guard against a null value by not calling init():
```

Comment:
Hm, are you sure this is needed? The `lint:ts` target and VSCode both don't complain if I remove this comment.

Reply:
Yeah, that's because I did not finish the typings in this file and thus did not add it to the `tsconfig.json`. I did so now in 254764a1b, though for our client-side partials, with all cajoling going on to obtain elements and string from a DOM that is disconnected from the script, and thus lots of non-null assertions being needed, I'm not sure how much value typing provides there.

Reply:
Oh that's right, I didn't add it to `tsconfig.json` because it references the `request-breach-scan` controller module, which references a couple of untyped modules and thus causes errors. 4ecc4ca1c